### DISCUSSION TITLE
Daofind  crashes with ERROR "pixel storage file is truncated "

### DISCUSSION THREAD

**Discussion by trappist1i** [iraf-community/iraf#210] [Q&A]
I have developed a photometry pipeline built on anaconda3/iraf27 that works on a Ubuntu x86_64 machine. The pipelines crashes from times to times with daofind producing the error: ERROR (807, "Pixel storage file is truncated (xxx.fits)"). 
The images for which it crashes do not seem to have anything special.
Any idea how to solve this annoying problem?

**Comment by olebole** [comment]
It is reproducible? I.e. can you provide a short script and an example file that crashes? And can you try to reproduce it with the current community IRAF release (2.17)?
